lib/common/output_xml.c

```diff
@@ -261,7 +261,7 @@ pcmk__xml_peek_parent(pcmk__output_t *out) {
     xml_private_t *priv = out->priv;
 
     CRM_ASSERT(priv != NULL);
-    CRM_ASSERT(g_queue_get_length(priv->parent_q) > 0);
```

Comment:
IMHO it's logical to allow peeking an empty stack. If not, I will bring this assertion back.